File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntPipelineVarMap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdPipelineVar;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioEntPipelineVarMap extends StudioBase
{
  public MetaIdPipelineVar[] keys;

  public Map<MetaIdPipelineVar, StudioEntPipelineVar> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntPipelineVarMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import java.util.Map
import com.neome.api.meta.base.Types.MetaIdPipelineVar
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioEntPipelineVar

interface StudioEntPipelineVarMap : StudioBase
{
  val keys: Array<MetaIdPipelineVar>
  val map: Map<MetaIdPipelineVar, StudioEntPipelineVar>
}